include/ebpf_nethooks.h

```diff
@@ -107,6 +107,10 @@ bind_hook_t(bind_md_t* context);
 //
 // CGROUP_SOCK_ADDR.
```

Comment:
Line 39 has xdp_hook_t.
Line 105 has bind_hook_t.
Where is the hook_t defined for this program type?